yt/yt/tools/import_table/lib/import_table.cpp

```diff
@@ -572,8 +595,10 @@ void ImportParquetFilesFromSource(
     auto writer = ytClient->CreateTableWriter<TNode>(metaInformationTable.Name());
     int fileIndex = 0;
     for (const auto& fileName : fileIds) {
-        writer->AddRow(TNode()(FileIdColumnName, fileName)(FileIndexColumnName, fileIndex));
-        ++fileIndex;
+        if (fileName.Contains(".parquet")) {
```

Comment:
Let's allow to configure it?

Reply:
Introduce a regexp in the configuration.